apps/frontend/src/views/Checklist.vue

```diff
@@ -0,0 +1,788 @@
+<template>
+  <Base
+    :show-search="true"
+    :title="curr_title"
+    @changed-files="evalInfo = null"
+  >
+    <!-- Topbar content - give it a search bar -->
+    <template #topbar-content>
+      <v-btn :disabled="!can_clear" @click="clear">
+        <span class="d-none d-md-inline pr-2"> Clear </span>
+        <v-icon>mdi-filter-remove</v-icon>
+      </v-btn>
+      <UploadButton />
+      <div class="text-center">
+        <v-menu>
+          <template #activator="{on, attrs}">
+            <v-btn v-bind="attrs" class="mr-2" v-on="on">
+              <span class="d-none d-md-inline mr-2"> Export </span>
+              <v-icon> mdi-file-export </v-icon>
+            </v-btn>
+          </template>
+          <v-list class="py-0">
+            <v-list-item class="px-0">
+              <v-tooltip top>
+                <template #activator="{on}">
+                  <IconLinkItem
+                    key="export_ckl"
+                    text="Export as CKL"
+                    icon="mdi-check-all"
+                    @click="export_ckl()"
+                    v-on="on"
+                  />
+                </template>
+                <span>JSON Download</span>
+              </v-tooltip>
+            </v-list-item>
+          </v-list>
+        </v-menu>
+      </div>
+    </template>
+    <template #main-content>
+      <v-container fluid grid-list-md pt-0 mt-4 mx-1>
+        <v-row>
+          <v-col md="4" :cols="12" class="pr-0 pl-1">
+            <!-- Data Table -->
+            <v-card height="94vh" overflow-auto>
+              <v-card-title class="pt-2">
+                <div
+                  style="
+                    display: flex;
+                    align-items: center;
+                    justify-content: space-between;
+                    width: 100%;
+                  "
+                >
+                  <div>
+                    <strong
+                      >Rules ({{ numItems }} shown,
+                      {{ loadedRules.length - numItems }} hidden)</strong
+                    >
+                  </div>
+                  <div style="width: fit-content">
+                    <v-switch
+                      v-model="shortIdEnabled"
+                      style="margin-top: 0; padding-top: 0"
+                      dense
+                      inset
+                      label="Short ID"
+                      color="teal"
+                      hide-details
+                    />
+                  </div>
+                </div>
+                <v-spacer class="mt-0 pt-0" />
+                <v-select
+                  v-model="selectedHeaders"
+                  :items="headersList"
+                  label="Select Columns"
+                  class="mt-4 pt-0"
+                  dark
+                  item-color="white"
+                  multiple
+                  outlined
+                  return-object
+                  height="8.5vh"
```

Comment:
should make a choice if we're doing responsive design or this type of 'vertical height' / absolute design

Reply:
Marking this change as resolved since it is a duplicate of another change request